static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -0,0 +1,171 @@
+@import 'svg';
+
+
+$contribute-yellow: #ffbb00;
+$gu-blue: #005689;
+$gu-blue-dark: #003d70;
+$comment-support-1: #7d7569;
+$white: #ffffff;
+$black: #000000;
+$gs-gutter: 20px;
+
+
+
+.interactive-embed {
+    height: 100%;
+    width: 100%;
+    border-top: 1px solid #4bc6df;
+    background-color: #efefec;
+
+    p {
+        line-height: 24px;
+        font-family: 'Guardian Egyptian Web';
```

Comment:
A couple of things throughout this file:

- we shouldn't be adding font-familes directly like this, we have have our [font mixins](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/typography/_src.scss#L68) - probably `fs-header` for this. 
- we should be trying to use the $gs-baseline and $gs-gutter variables for paddings and margin ensuring that our layouts are consistent.